q2_longitudinal/_utilities.py

```diff
@@ -518,6 +518,36 @@ def _visualize(output_dir, multiple_group_test=False, pairwise_tests=False,
     })
 
 
+def _visualize_anova(output_dir, pairwise_tests=False, model_results=False,
+                     residuals=False, pairwise_test_name='Pairwise t-tests'):
+    pd.set_option('display.max_colwidth', -1)
+
+    if pairwise_tests is not False:
```

Comment:
Can you add a unit test that looks at the results of the rendered HTML when this test is performed?

Reply:
I've added a test that loads the TSV (which is downloadable from the html). Good enough?